### PR TITLE
Top part looks a bit better

### PR DESCRIPTION
**Pull request by michaelghinrichs**
Really feeling the lack of scss and autoprefixer pain. Should add to webpack soon

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -1,3 +1,83 @@
...
+html {
+  box-sizing: border-box;
+}
+*, *:before, *:after {
+  box-sizing: inherit;
+}
+/* End Better Box sizing*/
+#main {
+	font-family: 'Roboto', sans-serif;
+	font-weight: 300;
```
Comment (awseward) [inline index.css]: Can we fix the indentation here plox?

File: index.html [hunk 2]
```diff
@@ -1,19 +1,38 @@
...
-      </button>
+		<div id="main">
+			<div class="tool-bar">
+				<span> width: </span>
+				<input type="text" placeholder="400">
+
+				<span> height: </span>
+				<input type="text" placeholder="300">
+
+				<span>actions: </span>
```
Comment (awseward) [inline index.html]: Can we clean up the indentation?

File: index.css [hunk 3]
```diff
@@ -1,3 +1,83 @@
...
+  font-family: sans-serif;
+  font-size: inherit;
+  color: inherit;
+  background: none;
+  cursor: pointer;
+  padding: 8px 10px;
+  display: inline-block;
+  /* margin: 15px 30px; */
+  text-transform: uppercase;
+	font-weight: 500;
```
Comment (awseward) [inline index.css]: Indentation plox

File: index.css [hunk 4]
```diff
@@ -1,3 +1,83 @@
...
+/* Button 1a */
+.btn-1:hover,
+.btn-1a:active {
+  color: #fff;
+  background: #333;
+}
+
+/* Action icons */
+.actions > * {
+	padding-left: 15px;
```
Comment (awseward) [inline index.css]: Indentation plox

**Comment by awseward** [comment]
+1 pending comments

File: index.css (hunk 1 above)
Comment (michaelghinrichs) [inline index.css]: so weird, i don't see any of this locally...

**Comment by awseward** [comment]
This is gonna take some followup work in the breadrepo once merged and "deployed" to production.

Hoping to get out of this weird 2-repo shuffle soon.

**Pull request merged**